src/Common/tests/System/Net/Http/Http2LoopbackConnection.cs

```diff
@@ -624,7 +633,42 @@ public async Task SendResponseBodyAsync(int streamId, ReadOnlyMemory<byte> respo
                 throw new Exception("Response body too long");
             }
 
-            await SendResponseDataAsync(streamId, responseBody, true).ConfigureAwait(false);
+            await SendResponseDataAsync(streamId, responseBody, isFinal).ConfigureAwait(false);
+        }
+
+        public override void Dispose()
+        {
+            WaitForConnectionShutdownAsync().GetAwaiter().GetResult();
```

Comment:
is this equivalent to just `.Wait()`?

Reply:
Not exactly.  It is better to use than `.Wait()` in many cases.  Using `.Wait()` will throw AggregateException with real exception inside it.  But using `.GetAwaiter().GetResult()` won't. It will only throw real exception.